File: Periode 3/Java/Chapter 1/3. Constructors and Collections/Exercise9/src/Sprint.java
import java.text.*;
import java.util.*;
import java.util.List;
import java.util.Scanner;

/**
 * Created by Joost on 23-2-2017.
 */
public class Sprint
{
    public List<UserStory> Userstory;
    public int HoursToDo;
    public int HoursDone;
    public int Duration;

    public Sprint(List<UserStory> userstory, int duration)
    {
        this.Userstory = userstory;
        this.HoursToDo = totalHours();
        this.HoursDone = 0;
        this.Duration = duration;
    }

    public int totalHours()
    {
        int output = 0;

        for(int i = 0; i < Userstory.size(); i++)
        {
            output += this.Userstory.get(i).Hours;
        }
        return output;
    }

    public String ToString()
    {
        return "";
    }

    public void NewUserStory()
    {
        Scanner in = new Scanner(System.in);

        System.out.println("Enter description");
        String d = in.nextLine();
        System.out.println("Enter hours");
        int h = in.nextInt();

        this.HoursToDo += h;
        this.Userstory.add(new UserStory(h, d));
    }

}

File: Periode 3/Chapter 1 - Done/3. Constructors and Collections - Done/Constructors.Collections/Constructors.Collections/Sprint.cs
using System;
using System.Collections.Generic;
using System.Linq;

namespace Constructors.Collections
{
    public class Sprint
    {
        public List<UserStory> UserStory;
        public string Duration;
        public DateTime StartDate;

        public Sprint(List<UserStory> userstory, string duration, DateTime date)
        {
            this.UserStory = userstory;
            this.Duration = duration;
            this.StartDate = date;
        }

        public string retStartdate()
        {
            return this.StartDate.ToString("dd-MM-yy");
        }

        public void NewUserstory()
        {
            Console.WriteLine("Voer beschrijving in: ");
            string description = Console.ReadLine();
            Console.WriteLine("Voer het aantal uren in: ");
            int hours = Convert.ToInt32(Console.ReadLine());

            UserStory.Add(new UserStory(hours, description));
        }

        public string HoursDone()
        {
            int output = 0;
            for (int i = 0; i < UserStory.Count; i++)
            {
                if (UserStory[i].Done == true)
                {
                    output += UserStory[i].Hours;
                }
            }

            return output + " hours are done";
        }

        public string hack()
        {
            for (int i = 0; i < UserStory.Count; i++)
            {
                UserStory[i].USDone();
            }
            return "Everything is done";
        }

        public string SprintDone()
        {
            int output = 0;
            for (int i = 0; i < UserStory.Count; i++)
            {
                if (UserStory[i].ToDo == true)
                {
                    output += UserStory[i].Hours;
                }
            }

            if(output == 0)
            {
                return "Yes the sprint is done";
            }
            else
            {
                return "No the sprint is not done";
            }
        }

        public string HoursRemain()
        {
            int output = 0;
            for (int i = 0; i < UserStory.Count; i++)
            {
                if (UserStory[i].ToDo == true)
                {
                    output += UserStory[i].Hours;
                }
            }

            return output + " hours must be realized.";
        }
    }
}